### ISSUE TITLE
Problem with dual screen

### ISSUE THREAD

**Issue by rememberYou** [swaywm/sway#1354]
Hello,

I'm having an issue with my dual screen. Indeed, I have two outputs: `eDP-1` and `DP-1`, and I would like to put the screen of my `e-DP1` output, right of the screen of my `DP-1`.

For that, I just added these two following lines in my `~/.config/sway/config` file:

```
output DP-1 pos 0,0 res 2560x1440
output eDP-1 pos 2560,0 res 2560x1440
``` 

After restarting sway, it's working until that I decide to change my background with ```swaymsg -t command "output * bg `/path/to/my/wallpaper` fill"```. In fact, the screen of my `eDP-1` output go back to the left of the screen of the `DP-1`.

Outputs of `swaymsg -t get_version`:

`sway version 0.14-rc1-47-ge12d1cf (2017-08-08, branch "master")`

Outputs of `swaymsg -t get_outputs`:

```
Output eDP-1
  Geometry: 2560x1440 @ 5120,0
  Scale factor: 1x
  Workspace: 1: 

Output DP-1
  Geometry: 2560x1440 @ 7680,0
  Scale factor: 1x
  Workspace: 2: 
```

To avoid overloading this issue, you can find with gists links:
1. [debug log](https://gist.github.com/231014bd7cdbc5ff66ff19481a4cad2a)
2. [my sway configuration](https://gist.github.com/rememberYou/11e8346f8c5cbda6d818b2f230060236)

Thank you for your help.

**Comment by SirCmpwn** [comment]
Move the bg line after your other output configs.

**Comment by rememberYou** [comment]
Hi SirCmpwn,

Thank you for your answer. I defined the ```output * bg `/path/to/my/wallpaper` fill``` after added the lines with `output DP-1` and `output e-DP1` but it didn't fix the problem.

I came up from the initial configuration that sway come up to reproduce this issue.

If I define the ```output * bg `/path/to/my/wallpaper` fill"``` line **before** the lines with `output DP-1` and `output e-DP1`, my `DP-1` screen is on the **left** and my `e-DP1` screen is on the **right**.

You can find the configuration to reproduce this [here](https://gist.github.com/rememberYou/6569a4b8c27ac947d432e089f804b5f0)

If now I define the ```output * bg `/path/to/my/wallpaper` fill"``` line **after** the lines with `output DP-1` and `output e-DP1`, my `DP-1` screen is on the **right** and my `e-DP1` screen is on the **left**.

You can find the configuration to reproduce this [here](https://gist.github.com/rememberYou/8d520d1cdf06380a4fe38c8bda66955d)

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unreleased sway 1.0.
